dist/index.js

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env node
+
+const Elm = require("./checker.js");
+const fs = require("fs");
+const minimist = require('minimist');
+const path = require('path');
+
+const helpStr =
+    `Usages:
+$ elm-docstyle [elm_code_directory | path_to_elm_file]
+# Docstyle the project and log messages to the console.
+
+Options:
+    --help, -h          Print the help output.
+    --version, -v       Print the package version.
+    --verbose           If set, the offending comments are added to the error report in full.
+    --check_all         If set, the checker goes through all declarations; otherwise, only through exported ones.
+    --config_path       Path to the elm-docstyle JSON config. If unspecified, a default config is used.
+    --format            Output format for CLI. Defaults to "human". Options "human"|"json".`;
+
+
+const args = minimist(process.argv.slice(2), {
+    alias: {
+        help: 'h',
+        version: 'v'
+    },
+    boolean: ['help', 'version', 'verbose', 'check_all'],
+    string: ['config_path']
+});
+
+(function () {
+    const elmDocstyleVersion = require(path.join(__dirname, '../', 'package.json')).version;
+    if (args.help) {
+        console.log(helpStr);
+        process.exit(1);
+    }
+
+    if (args._.length === 0) {
+        console.log("Please specify at least one directory or path to Elm source file.\n");
```

Comment:
Please join the console.log calls into one function call. While the default logger might be still readable, it can become confusing when the messages are separated which actually belong together.

(Please also fix the console.log(.) calls below)